sunpy/time/time.py

```diff
@@ -212,22 +188,25 @@ def convert_time_str(time_string, **kwargs):
                 break
             if ts is None:
                 continue
-            return datetime.strptime(ts, time_format) + time_delta
+            return ap.Time.strptime(ts, time_format,
+                                    **kwargs) + time_delta
         except ValueError:
             pass
     time_string_parse_format = kwargs.pop('_time_string_parse_format', None)
```

Comment:
This is probably best done as a followup PR, but I think I would like to remove this functionality from `parse_time`.